magellan-library/src/main/java/com/wealthfront/magellan/lifecycle/ShownFieldFromContext.kt

```diff
@@ -0,0 +1,22 @@
+package com.wealthfront.magellan.lifecycle
+
+import android.content.Context
+
+public class ShownFieldFromContext<Field>(
```

Comment:
If we end up renaming the method associated with this, let's rename this class too